.buildkite/pipeline.yml

```diff
@@ -1,14 +1,4 @@
 x-common-params:
-  - &gb-mobile-docker-container
```

Comment:
I removed this common parameter as it's no longer used in the pipeline configuration.